draft-ietf-mls-protocol.md

```diff
@@ -1556,11 +1559,13 @@ struct {
     MLSMessageContent content;
     select (MLSMessageContentTBS.content.sender.sender_type) {
         case member:
-        case new_member:
+        case new_commit_member:
             GroupContext context;
-
         case external:
             struct{};
+        case new_proposal_member:
+            // The KeyPackage included in embedded Add proposal
+            KeyPackage key_package;
```

Comment:
This doesn't seem all that useful, since the KeyPackage is already being signed via the Add proposal.  I would just make this `struct{}`.